src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -159,8 +159,6 @@ public String getApiUri() {
         } catch (RateLimitExceededException rle) {
             listener.getLogger().format("%n%s%n%n", rle.getMessage());
             throw new InterruptedException();
-        } catch (IOException e) {
-            // may be a user... ok to ignore
```

Comment:
The real problem was [here](https://github.com/recena/cloudbees-folder-plugin/blob/b4f2bffd2659a6004bf932553f0b3059ce5ff1c0/src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java#L194-194). If `computeChild()` finishes through a `continue` sentence, `orphaned` list will have all `items` because none was omitted.